accel-pppd/ifcfg.c

```diff
@@ -334,3 +342,51 @@ int __export ap_session_rename(struct ap_session *ses, const char *ifname, int l
 	return 0;
 }
 
+int __export ap_session_set_vrf(struct ap_session *ses)
```

Comment:
I think ap_session_set_vrf(struct ap_session *ses, const char *vrf_name, int len) will be enough here, NULL can be used for default (none) vrf.
On success, func will allocate and fill ses->vtf_ifname for possible reference.
Also, it will be checked and recovered in asp_session_finished()